ansible/roles/tilemaker/templates/bicycle/process-openmaptiles.lua

```diff
@@ -655,6 +655,11 @@ function WritePOI(obj,class,subclass,rank)
 		end
 		obj:Attribute("bike", isBike)
 	end
+
+	if class=="leisure" and subclass=="track" then
+		local sportType = obj:Find("sport")
+		obj:Attribute("sport", sportType)
```

Comment:
Instead of adding this to the poi laye we can simplify the styling and combine tracks and pitches in the same class.

Can you try the following?

```suggestion
                obj.Layer("landuse", false)
		obj:Attribute("class", "pitch")
```

That way we don't need to have two style definitions.